src/stories/Library/Forms/checkbox/checkbox.scss

```diff
@@ -22,7 +22,7 @@
   transform: translate3d(0, 0, 0);
 }
 
-.checkbox__label .checkbox__icon:first-child {
+.checkbox__label .checkbox__icon:first-of-type {
```

Comment:
Consider whether this change should be included.

Personally I do not think it makes much sense:

1. Googling has not lead me to a reason to why using `:first-child` is a problem for us.
2. We might remove it in `checkbox.scss` but nothing prevents us from adding it elsewhere tomorrow. If we want to avoid using `:first-child` altogether we should update our Stylelint configuration to raise errors for `:first-child`.
3. The change it not relevant to the issue at hand.

Reply:
I understand your thinking, but I really wanted to get rid of this nasty red error from the console:
![image](https://user-images.githubusercontent.com/28546954/182197937-76e3c55c-5b19-4a56-9d82-411a7cdada81.png)